apps/ui/src/Pages/RidesFeed/CreateRideDialog.tsx

```diff
@@ -168,14 +188,17 @@ const CreateRideDialog = ({
           type="number"
           fullWidth
           value={seats}
-          onChange={(e) => setSeats(e.target.value)}
+          onChange={(e) => handleGasSeatsChange('seats', e.target.value)}
+          error={isSeatsLessThanOne || isSeatsNotInteger}
+          helperText={isSeatsLessThanOne ? 'Seats must be 1 or more' : isSeatsNotInteger
+            ? 'Seats must be complete' : '' }
         />
```

Comment:
use consts instead

Reply:
simple sentence, decided to leave it here